libsolidity/parsing/Parser.cpp

```diff
@@ -1986,23 +1985,34 @@ pair<vector<ASTPointer<Expression>>, vector<ASTPointer<ASTString>>> Parser::pars
 	return ret;
 }
 
-optional<string> Parser::findLicenseString()
+optional<string> Parser::findLicenseString(std::vector<ASTPointer<ASTNode>> const& _nodes)
 {
 	// We circumvent the scanner here, because it skips non-docstring comments.
-	static regex const licenseRegex("SPDX-License-Identifier:\\s+([^\\n\\r]+)");
-
+	static regex const licenseRegex("SPDX-License-Identifier:\\s*([^\\n\\r\\s]+)");
```

Comment:
I think we cannot exclude spaces here because of license expressions.

Reply:
Yes, thats true. I didn't thought about these expressions.